modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/IncidentContactCard/IncidentContactCard.jsx

```diff
@@ -136,155 +139,171 @@ const IncidentContactCard = ({accountSubscriptionGroupsNames}) => {
 	);
 
 	return (
-		<div
-			className={`customer-portal-card-footer ${
-				isLXCEnvironment
-					? 'customer-portal-card-footer-style-lxc'
-					: 'customer-portal-card-footer-style-ac'
-			}`}
-		>
-			<div className="customer-portal-card-footer-title">
-				<h1>{i18n.translate('incident-contacts')}</h1>
-			</div>
-			<>
-				<div className="customer-portal-card-footer-description">
-					<p>
-						{i18n.translate(
-							'team-members-who-can-be-contacted-with-high-priority-messages'
-						)}
-					</p>
-				</div>
+		<>
+			{hasActiveProduct && (
+				<>
+					<div
+						className={`customer-portal-card-footer ${
+							isLXCEnvironment
+								? 'customer-portal-card-footer-style-lxc'
+								: 'customer-portal-card-footer-style-ac'
+						}`}
```

Comment:
Use classnames for conditional class mapping

```suggestion
						className={classNames('customer-portal-card-footer', {
						     'customer-portal-card-footer-style-lxc': isLXCEnvironment,
						      'customer-portal-card-footer-style-ac': !isLXCEnvironment,
						})}
```

It's necessary to import

`import classNames from 'classnames'`

Reply:
Done!